code/modules/library/lib_items.dm

```diff
@@ -23,6 +23,9 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 0)
 	var/state = 0
 	var/list/allowed_books = list(/obj/item/book, /obj/item/spellbook, /obj/item/storage/book) //Things allowed in the bookcase
+	var/load_random_books = FALSE
+	var/random_category = null
+	var/books_to_load = 0

```

Comment:
[Please add DMDOC which matches the required standard](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
Technically this isn't new code ( ͡° ͜ʖ ͡°)